src/utils/AppConsts.ts

```diff
@@ -27,24 +27,19 @@ export const MAXIMUM_ZOOM_SCALE = 12;
 export const ZOOM_INCREASE = 1.25;
 export const ZOOM_DECREASE = 0.8;
 
-export enum LabelColors {
-    PROVIDER = '#00bfc2',
-    CONSUMER = '#039881',
-    PROVIDER_CONSUMER = '#03939f',
-    DEFAULT = '#dcdee0',
-    FOCUSED = '#4ca3e5',
-}
+export enum Colors {
+    WHITE = '#FFFFFF',
```

Comment:
Maybe we should use HL names?

Reply:
changed color names that resembled ones from our brand, left others unchanged. Is that satisfying?